src/main/webapp/permissions-manage-files.xhtml

```diff
@@ -314,7 +321,9 @@
                                     <p:ajax event="toggleSelect"  update=":rolesPermissionsForm:selectionCounter" />
 
                                     <p:column selectionMode="multiple" style="width:2%;text-align:center"/>
-                                    <p:column headerText="#{bundle['dataverse.permissionsFiles.assignDialog.fileName']}">
+                                    <p:column width="98%"  headerText="#{bundle['dataverse.permissionsFiles.assignDialog.fileName']}">
+                                        <h:outputText value=" (#{bundle['dataverse.permissionsFiles.files.deleted']}) " rendered="#{file.deleted}"/>
+                                        <h:outputText rendered="#{!empty file.directoryLabel}"  value="#{file.directoryLabel}/"/>
```

Comment:
See comment above at line 236 about the file.directoryLabel, in the Grant File Access popup, looking out of order because the table rows are sorted by File Name. Same applies here to the Restricted Files table.